packages/plugin-e2e/package.json

```diff
@@ -32,7 +32,7 @@
     "playwright:all": "npx playwright test"
   },
   "engines": {
-    "node": ">=18 <=20"
+    "node": ">=18"
```

Comment:
Any particular reason to drop the upper limit? Does this suggest the package will work with Node 23 too?

Reply:
I think it would. But this is a good point. I'm rather safe than sorry. Changing to `>=18 <=22`